gnocchi/rest/api.py

```diff
@@ -1858,6 +1868,18 @@ def get_cross_metric_measures_from_objs(metrics, start=None, stop=None,
                          for metric in metrics),
                     'No granularity match'))
 
+        aggregations = set()
+        for metric in metrics:
+            for g in granularity:
+                agg = metric.archive_policy.get_aggregation(
+                    aggregation, g)
+                if agg is None:
+                    abort(404, six.text_type(
+                        storage.AggregationDoesNotExist(metric, aggregation, g)
+                    ))
+                aggregations.add(agg)
+        aggregations = sorted(aggregations, reverse=True)
```

Comment:
shouldn't this be sorted on a key?

Reply:
It's sorted on granularity because it's the second field and the first one is the same (the method), but I guess it's not explicit. :) fixed